tests/Validity.hs

```diff
@@ -89,8 +108,8 @@ instance (Eq k, Hashable k, Validity k, Validity v) =>
                                             check
                                             "The hash, when masked with the current bitmask, of the key, produces the index in the array where the hashmap is found" $
                                         let h = HM.hashWithSalt s k
-                                        in HM.sparseIndex bm (HM.mask h bs) ==
-                                           ix
+                                         in HM.sparseIndex bm (HM.mask h bs) ==
+                                            ix
```

Comment:
Could you indent the `let` and `in` a couple spaces and line them up with each other? I find this hard to read.